lib/resources/r4/appointment_patch.yaml

```diff
@@ -40,6 +40,26 @@ operations:
         <li>When present, this patch operation must be accompanied by the replace Status patch operation with a value of "booked"</li>
         <li>Only one Slot reference may be provided in the value array</li>
       </ul>
+  - name: add-reasonCode
+    path: /reasonCode
+    operation: add
+    type: List of CodeableConcept
+    url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.reasonCode
+    description: The clinical reason for why this appointment is being scheduled
+    example: |
+      {
+        "op": "add",
+        "path": "/reasonCode",
+        "value": [
+          {
+            "text": "I have a headache"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>This patch operation is only supported when updating the Appointment.status with a value of "booked"</li>
+      </ul>
```

Comment:
add note indicating only one codeable concept is allowed

```
Only one CodeableConcept may be provided in the value array
```

Reply:
[Added and fixed a typo](https://github.com/cerner/fhir.cerner.com/commit/3d1e971d35ff233b93fb577f25faa807df709823#diff-07dbf4d3264a1e7a98f8d854b77f88e9R62)